chia/wallet/puzzles/p2_singleton_via_delegated_puzzle_safe.clsp

```diff
@@ -27,7 +27,10 @@
     )
     (c
       (list ASSERT_MY_COIN_ID my_id)
-      (a delegated_puzzle delegated_solution)
+      (c
+        (list CREATE_COIN_ANNOUNCEMENT '$')
```

Comment:
The `'$'` is no longer necessary.  We used to use it because we didn't know how to use python very well but now we've learned better.